content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+
+Looking again at the button HTML code, by using `on="tap:amp-access.login-sign-in"`, we specify that once the user taps on the button, the URL specified in the JSON above should be used:
+
+[sourcecode:json]
+{ 
+	"login": {
+          "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL"
+  }
+}
+
+[/sourcecode]
+
+Notice that it’s possible to define different URLs inside the login node, in this case we are defining `sign-in`, we will later define `sign-out`
+
+The login page is a normal HTML page in which we populate the login and password values for the sake of simplicity. Notice the usage of `returnURL` hidden input type, which is populated by the AMPByExample server via server-side templating. The server reads this value from a parameter called `return` automatically added by the AMP library to the sign-in URL. 
+
+Find below an example of the value for the `return` parameter added to the request once you click on the login button, you can explore this value by using the Chrome dev console navigating to the Network tab.
+
+<amp-img src="/static/img/return-parameter.png" alt="Return parameter" height="150" width="600"></amp-img>
+
+
+Once the AMPByExample server receives the post request from the login page and the login and password are correct, it redirect to the `returnURL` we mentioned above appending the parameter `#success=true`. The AMP runtime can now authorize the page and finally allow you to add a comment.
+
+It’s important to understand what the AMP runtime does and what the server should be doing, as the implementation of the server is responsibility of the publisher of the page. 
+
+As a quick recap:
+
+- The AMP runtime automatically adds the return parameter to the sign-in request specified inside the login JSON object 
+-  The AMP runtime understands that should should close the login page and redirect to the page specified by the return URL parameter. 
+-  The server should orchestrate the response once the user clicks on the login button
+
+
+A more detailed explanation about this flow can also be found [here](https://www.ampproject.org/docs/reference/components/amp-access#login-flow). 
+
+## Add a comment
+
+<amp-img src="/static/img/comment.png" alt="Add comment" height="325" width="300"></amp-img>
+
+At this point, you can add a comment using the `amp-form` library. Notice how the presence of the form is conditioned by the `amp-access` component:
```

Comment:
Not sure if conditioned is the right word here.. (and above as well). Maybe change to something like "Notice how the presence of the form is conditional, depending on the state `amp-access` component:"